### PR TITLE
show custom error page for csrf error

### PR DESCRIPTION
**Pull request by Ntu2koTrevor**
@codiebeulaine Please review cc @saeedmb8 
screen shot 2016-10-19 at 1 12 24 pm

### CODE REVIEW

File: molo/core/tests/test_views.py [hunk 1]
```diff
@@ -312,6 +312,7 @@ def test_featured_topic_of_the_day(self):
             demote_date=demote_date
         )
         response = self.client.get('/')
+        print response
```
Comment (codiebeulaine) [inline molo/core/tests/test_views.py]: Stray Print

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/403_csrf.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+{% block body_class %}default-container{% endblock %}
+
+{% block content %}
+<div class="container">
+    <div class="title">
+        <h3>{% trans "Unable to Login" %}</h3>
+    </div>
+    <div class="message-notification">
+        {% trans "Hi" %},
+        <p>{% trans "Sorry, you are having trouble accessing this feature! Due to a temporary technical issue, log in and registrations are not available on the Free Basics app for now. To login or register for a profile, visit our free mobisite through your mobile browser at" %} <a href="http://0.freebasics.com/" target="_blank">0.freebasics.com/</a>.{% trans " We're working hard to get this problem resolved. Thank you." %} <br/>
```
Comment (codiebeulaine) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/403_csrf.html]: is this the website that they should be going to?

**Comment by codiebeulaine** [comment]
👍

**Comment by Ntu2koTrevor** [comment]
@saeedmb8 Ready for re-review

**Comment by saeedmb8** [comment]
👍  🍰

**Pull request merged**